### ISSUE TITLE
Can't select from seeks tab after finishing a FICS game 

### ISSUE THREAD

**Issue by pychess** [pychess/pychess#224]
Original [issue 224](https://code.google.com/p/pychess/issues/detail?id=224) reported by [gbtami](https://code.google.com/u/gbtami/) 2007-09-27

**What steps will reproduce the problem?**
1. Logged in to FICS with my account
2. played a rated game, selected from seeks tab
3. After the game finished with mate, wanted to select another one to play
again (with mouse), but all games in seeks tab was disabled.

**Comment by pychess** [comment]
Original [comment 1.](https://code.google.com/p/pychess/issues/detail?id=224#c1) posted by [lobais](https://code.google.com/u/lobais/) 2007-09-28

I'm unable to reproduce this.
Can you still do so?
- added label: Release0.8

**Comment by pychess** [comment]
Original [comment 2.](https://code.google.com/p/pychess/issues/detail?id=224#c2) posted by [gbtami](https://code.google.com/u/gbtami/) 2007-09-28

Yes.
With Guest login there is no problem.
Have you tried with your account?

**Comment by pychess** [comment]
Original [comment 3.](https://code.google.com/p/pychess/issues/detail?id=224#c3) posted by [lobais](https://code.google.com/u/lobais/) 2007-09-30

Yes. I tried exactly as you said.

Actually, what did you mean by "all games in seeks tab was disabled"? Were there
actually games in the list? It should have been empty.

**Comment by pychess** [comment]
Original [comment 4.](https://code.google.com/p/pychess/issues/detail?id=224#c4) posted by [gbtami](https://code.google.com/u/gbtami/) 2007-09-30

No. After finished the game, the seek list start to grow/shrink again, but i can't
select from the list a new seek to play with.

**Comment by pychess** [comment]
Original [comment 5.](https://code.google.com/p/pychess/issues/detail?id=224#c5) posted by [lobais](https://code.google.com/u/lobais/) 2007-10-06

Can you still replace this everytime in svn?
I haven't been able to do so a single time.
Can you send the log from one of the times it happens?

**Comment by pychess** [comment]
Original [comment 6.](https://code.google.com/p/pychess/issues/detail?id=224#c6) posted by [gbtami](https://code.google.com/u/gbtami/) 2007-10-07

svn r682
Tried to log in with my account, but got this:
tamas@tami:~/svn/pychess$ p
/home/tamas/svn/pychess/lib/pychess/Main.py:478: PangoWarning: Invalid UTF-8 string
passed to pango_layout_set_text()
  gtk.main()
Traceback (most recent call last):
  File "/home/tamas/svn/pychess/lib/pychess/ic/ICLogon.py", line 120, in onConnected
    self.lounge = ICLounge(connection)
  File "/home/tamas/svn/pychess/lib/pychess/ic/ICLounge.py", line 36, in __init__
    sections = (
  File "/home/tamas/svn/pychess/lib/pychess/ic/ICLounge.py", line 310, in __init__
    ParrentListSection.__init__(self)
  File "/home/tamas/svn/pychess/lib/pychess/ic/ICLounge.py", line 266, in __init__
    self.listPublisher = Publisher(updateLists, Publisher.SEND_LIST)
  File "/home/tamas/svn/pychess/lib/pychess/System/GtkWorker.py", line 23, in __init__
    Thread.__init__(self)
TypeError: unbound method __init__() must be called with Thread instance as first
argument (got Publisher instance instead)

**Comment by pychess** [comment]
Original [comment 7.](https://code.google.com/p/pychess/issues/detail?id=224#c7) posted by [gbtami](https://code.google.com/u/gbtami/) 2007-10-07

Played a rated game (10 gbtami vs. Brunek).
- attached [win.pgn](https://code.google.com/p/pychess/issues/detail?id=224#c7)
- attached [2007-10-07_17-55-58.log](https://code.google.com/p/pychess/issues/detail?id=224#c7)

**Comment by pychess** [comment]
Original [comment 8.](https://code.google.com/p/pychess/issues/detail?id=224#c8) posted by [lobais](https://code.google.com/u/lobais/) 2007-10-07

Congrats with the victory.
I don't see anything suspecious in the game, so I've commited some debugging prints
which I hope will pinpoint us to the problem.
Just to be sure: You can't reproduce this in unrated games?

**Comment by pychess** [comment]
Original [comment 9.](https://code.google.com/p/pychess/issues/detail?id=224#c9) posted by [gbtami](https://code.google.com/u/gbtami/) 2007-10-08

Now played an unrated game (gbtami vs zzzzzztrainer), but seekpanel doesn't refreshed
just after i closed the game. I was waiting at least 1 or 2 min.
- attached [2007-10-08_10-08-35.log](https://code.google.com/p/pychess/issues/detail?id=224#c9)
- attached [unrated-game.png](https://code.google.com/p/pychess/issues/detail?id=224#c9)

**Comment by pychess** [comment]
Original [comment 10.](https://code.google.com/p/pychess/issues/detail?id=224#c10) posted by [lobais](https://code.google.com/u/lobais/) 2007-10-12

This is so strange. We can read out of your logfile, that set_sensitive(True) is
actually called, but as we can see on the picture, no change is made.
Perhaps it is a bug in your gtk version..
We could try to set sensitive true for all children of "seekListContent".
Btw. does the challengepanel, the seekgraph and the seekpanel also remain insensitive?

**Comment by pychess** [comment]
Original [comment 11.](https://code.google.com/p/pychess/issues/detail?id=224#c11) posted by [gbtami](https://code.google.com/u/gbtami/) 2007-10-24

Looking in the log again, i have to say the pb not related to set_sensitive, as it
happened after i closed the game. The screenshot was created when i did my last move
and after i waited minutes.
Seems my last mating move (Vh8#) was not sended to FICS or what.

**Comment by pychess** [comment]
Original [comment 12.](https://code.google.com/p/pychess/issues/detail?id=224#c12) posted by [lobais](https://code.google.com/u/lobais/) 2007-10-28

I tracked it down to be a bug in the telnet communication code.
Two different managers were connected to the "Game has ended" line. The BoardManager
used it to end the game, the GameListManager used it to add the two players to the
players list.
Only one would have it.

Should be fixed in 731
- changed status to: Fixed
